test-app/src/main/res/values/arrays.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string-array name="documentSelectorArray">
-        <item>Copy to app storage</item>
+        <item>Import to app storage</item>
```

Comment:
I think `Copy` was more consistent with the other options. All of them are **importing** publications, but the verb explains the way the publication file is accessed.

Reply:
I introduced a subtle distinction in the `Bookshelf` that is reflected here: you can _add_ an asset to the database, or _import_ a publication from it to the app storage. In the latter case but not in the former, LCP publications will be downloaded and stored locally.